src/Components/Footer/Social.js

```diff
@@ -8,7 +8,7 @@ function Social() {
       <Text style={[styles.text, styles.heading]}>Follow us here</Text>
       <View style={styles.socialContainer}>
         <TouchableHighlight
-          onPress={() => Linking.openURL('https://twitter.com/AnitaB_org')}
+          onPress={() => window.open('https://twitter.com/AnitaB_org','_blank')}
```

Comment:
```suggestion
          onPress = {() => window.open('https://twitter.com/AnitaB_org', '_blank')}
```
Same changes for other lines.

Reply:
@codesankalp I have done the suggested changes.